Makefile

```diff
@@ -120,6 +120,18 @@ clean-hax:
 	         rm -rf $$d; \
 	     fi; \
 	 done
+	@if [[ -d hax/hax/motr/__pycache__ ]]; then \
```

Comment:
Can't we include this directory into `for` loop above? I don't think it is worth copying and pasting the same pieces of code.

Reply:
Done the changes suggested.